crates/sui-protocol-config/src/lib.rs

```diff
@@ -454,6 +524,17 @@ impl ProtocolConfig {
         // To change the values here you must create a new protocol version with the new values!
         match version.0 {
             1 => Self {
+                max_tx_in_batch: Some(10),
```

Comment:
Is this still relevant/used now that we have `max_programmable_tx_commands` (i.e., are batches dead)?

Reply:
Would you like me to remove this limit in anticipation of removal of batches, or keep it here for now? I don't have a strong opinion on this. cc @tnowacki